Source/WebCore/html/HTMLBodyElement.idl

```diff
@@ -29,15 +29,14 @@
     [CEReactions=NotNeeded, Reflect] attribute [LegacyNullToEmptyString] DOMString text;
     [CEReactions=NotNeeded, Reflect] attribute [LegacyNullToEmptyString] DOMString vLink;
 
-    // Non-standard
```

Comment:
We probably want to keep this comment for the `onwebkit*` handlers below. And perhaps for focusin/out we could refer to the GitHub issue, though it's a somewhat widely known problem so maybe that's okay.